editor/inserter/menu.js

```diff
@@ -225,7 +225,15 @@ class InserterMenu extends wp.element.Component {
 	render() {
 		const { position = 'top', instanceId } = this.props;
 		const visibleBlocksByCategory = this.getVisibleBlocksByCategory( wp.blocks.getBlockTypes() );
-		const positionClasses = position.split( ' ' ).map( ( pos ) => `is-${ pos }` );
+		const visualEditorHeight = document.querySelector( '.editor-visual-editor' ).clientHeight;
```

Comment:
Ideally a component would not need to make any assumptions about the presence of any other elements on the page. In this case, it's both a little bit more reasonable because it's at least contained within `editor`-specific bundle, not a generic component, and because it's a difficult problem to solve (similar to in #1193 where I'd remarked about issue of knowing bounds of editor canvas).